tests/org.jboss.reddeer.eclipse.test/src/org/jboss/reddeer/eclipse/test/m2e/core/ui/preferences/MavenErrorPreferencePageTest.java

```diff
@@ -0,0 +1,82 @@
+/******************************************************************************* 
+ * Copyright (c) 2016 Red Hat, Inc. 
+ * Distributed under license by Red Hat, Inc. All rights reserved. 
+ * This program is made available under the terms of the 
+ * Eclipse Public License v1.0 which accompanies this distribution, 
+ * and is available at http://www.eclipse.org/legal/epl-v10.html 
+ * 
+ * Contributors: 
+ * Red Hat, Inc. - initial API and implementation 
+ ******************************************************************************/ 
+package org.jboss.reddeer.eclipse.test.m2e.core.ui.preferences;
+
+import static org.junit.Assert.assertTrue;
+
+import org.jboss.reddeer.eclipse.m2e.core.ui.preferences.MavenErrorPreferencePage;
+import org.jboss.reddeer.junit.runner.RedDeerSuite;
+import org.jboss.reddeer.swt.exception.SWTLayerException;
+import org.jboss.reddeer.workbench.ui.dialogs.WorkbenchPreferenceDialog;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * Test for MavenErrorPreferencePage
+ * 
+ * @author ldimaggi
+ *
+ */
+
+@RunWith(RedDeerSuite.class)
+public class MavenErrorPreferencePageTest {
+
+	private WorkbenchPreferenceDialog preferencesDialog = new WorkbenchPreferenceDialog();
+	
+	private MavenErrorPreferencePage mavenPreferencePage = new MavenErrorPreferencePage();
+
+	@Test
+	public void checkAllPreferences() {
+		preferencesDialog.open();
+		preferencesDialog.select(mavenPreferencePage);
+		
+		/* Verify default values */		
+		assertTrue(mavenPreferencePage.getGroupId().equals("Warning"));
```

Comment:
asserting default values does not make sense. If Eclipse decide to change it later, it will fail on this place but it is not a bug in our code, only tests are not robust enough. Instead of it, I would obtain default values and try to change them to a different one and assert that change has been done.